warl_proposal/csr_template_draft1.yaml

```diff
@@ -0,0 +1,130 @@
+############################################ Template description ############################################
```

Comment:
One thing that's not defined in the template is the shadow register implementation. For example, write to MSTATUS.SPP could also change SSTATUS.SPP. We should have some way to describe this relationship.

Reply:
hmm.. interesting.. I might have overlooked that relationship. 
From what I understand from the spec: the WARL functions of updating mstatus.spp and sstatus.spp can be different, so status.spp has to exist as a separate node. 

What we could do is provide another node in the <subfield> yaml to indicate what this field shadows

```
shadow: csr_name::subfield_name

e.g.:

status:
....
  spp:
    msb: 8
    lsb: 8
    shadow: mstatus::spp

```
Note: I am using `::` as accessing delimeter.

Reply:
Looks good, thanks

Reply:
unresolving this for now. 
Once I update the doc with this new-field, I will mark this resolved.